plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcMetadataConfig.java

```diff
@@ -107,4 +110,18 @@ public JdbcMetadataConfig setDomainCompactionThreshold(int domainCompactionThres
         this.domainCompactionThreshold = domainCompactionThreshold;
         return this;
     }
+
+    @Positive
+    public int getInsertBatchSize()
+    {
+        return insertBatchSize;
+    }
+
+    @Config("insert.batch-size")
+    @ConfigDescription("Maximum number of rows to insert in a single batch")
```

Comment:
Add validation: `@Min(1) @Max(someSaneValue)`

Reply:
What that someSaveValue might be?

Reply:
Picking 1_000_000, seems sane enough to me.